backend/petshop/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.micifuz</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <artifactId>micifuz-petshop</artifactId>
+    <packaging>jar</packaging>
+    <name>Micifuz: petshop</name>
+    <dependencies>
+        <dependency>
+            <groupId>com.micifuz</groupId>
+            <artifactId>commons</artifactId>
+        </dependency>
+        <!-- Test resources -->
+        <dependency>
+            <groupId>com.micifuz</groupId>
+            <artifactId>test-resources</artifactId>
```

Comment:
Should test-resources be  under test scope?

Reply:
the scope has moved to dependency management

```
<dependency>
                <groupId>com.micifuz</groupId>
                <artifactId>test-resources</artifactId>
                <version>${project.version}</version>
                <scope>test</scope>
            </dependency>
```